chamber-of-commerce.md

```diff
@@ -0,0 +1,93 @@
+### 3. Review Process
+3.1 The Chamber of Commerce will review each proposal to ensure it meets the following criteria:
+   a) Fits within the theme and setting of a university town in North Carolina
+   b) Does not significantly overlap with existing businesses in Woodcrest
+   c) Provides roleplay opportunities for the community
+   d) Complies with all sim rules and regulations
+3.2 The Chamber may request additional information or suggest changes to the proposal before making a decision.
+3.3 Proposals will be approved or denied within one week of submission.
+
+### 4. Post-Approval Steps
+4.1 Upon approval, individuals must complete the rental process for their designated commercial plot within one week.
+4.2 Individuals must open their business within one week of renting the plot.
+4.3 Businesses must adhere to all sim rules and regulations, including any specific to commercial properties.
+4.4 The Chamber of Commerce reserves the right to revoke approval if the business deviates significantly from the approved proposal or violates any rules.
+
+### 5. Changes and Transfers
+5.1 Any significant changes to the business (e.g. change of location, major shift in products/services) must be submitted to the Chamber for approval.
+5.2 If an individual wishes to transfer ownership of their business to another individual, the new owner must submit a proposal to the Chamber for approval.
+
+### 6. Sim-Owned Businesses
+6.1 Businesses that are owned and operated by the sim administrators (i.e. not individually-owned) will be exempt from the application and approval process.
+6.2 These businesses will be carefully selected to enhance the overall roleplay experience and fill any gaps in the Woodcrest business ecosystem.
+6.3 Sim-owned businesses will still be required to adhere to all relevant rules and regulations.
+
+### 7. Penalties for Non-Compliance
+7.1 Businesses that deviate from their approved proposal or violate any Woodcrest or North Carolina laws will face penalties, which may include:
+   a) Fines or civil penalties
+   b) Temporary closure or suspension of operations
+   c) Eviction from the commercial rental plot
+   d) In severe cases, forfeiture of any rent paid and permanent ban from operating a business in Woodcrest
+7.2 The specific penalties will be determined by the Chamber of Commerce and/or sim administrators based on the nature and severity of the violation.
+7.3 Individuals will have the opportunity to appeal any penalties through a formal process outlined by the Chamber.
+
+## Required Permits and Licenses
+The following types of businesses will require additional permits or licenses to operate in Woodcrest:
+- Restaurants and food service: Health department certification, food handler's permits for all employees
+- Bars and nightclubs: Alcohol permit from the North Carolina ABC Commission
+- Cannabis dispensaries: Dispensary license from the North Carolina Department of Health, proof of compliance with all state regulations
```

Comment:
This is a good topic to discuss for sure and agree that this should be a tangible object that can be given to individuals and be displayed in the place of business.

Reply:
Do we want to include anything here that people can potentially try and open businesses illegally (i.e. a food truck that serves cannabis) but they may have to face IC consequences and be told to stop it from Moderators.Maybe a "Businesses" section in the code of conduct or in variances? 

I would expect this to only be allowed by seasoned RPers so would we want to include this option in the application, at risk of encouraging it.

Is this something we want to even try? I can see it potentially being difficult to manage. I don't feel particular strong either way about this

Reply:
I absolutely don't want to discourage illegal activity, just I don't think that putting it in this document is the right place for it.  Since this is really intended on being a gating mechanism for business rentals, putting something roleplay specific like that feels it would be more appropriate in one of the roleplay covenant documents.